docs/reach/common/tutorials/placing-gcps.md

```diff
@@ -25,7 +25,7 @@ To place GCPs with Reach units follow the steps from this guide.
 	</center>
 
 ??? note "Setting RTK on Reach rover over NTRIP"
-	If you have just one Reach receiver, you need to connect it to NTRIP first. [Check this guide for more details](../../../ntrip-workflow)
+	If you have just one Reach receiver, you need to connect it to NTRIP first. [Check this guide for more details](https://docs.emlid.com/reachrs/common/quickstart/ntrip-workflow/)
```

Comment:
Please check how we specify links in other docs. We don't use direct links.

Reply:
Changed to (../../quickstart/ntrip-workflow/)

Checked on the localhost. Works fine.